storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Channels/CacheChannel.cs

```diff
@@ -0,0 +1,62 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+using Dotnet.Storm.Adapter.Messaging;
+
+namespace Dotnet.Storm.Adapter.Channels
+{
+    internal class CacheChannel : Channel
+    {
+        private static Queue<Message> Cache = new Queue<Message>();
```

Comment:
Updated based on our discussion

Reply:
Is Queue thread safe?

Reply:
No. According to discussion with Yuriy, the idea of the test API in adapter only support testing a single component by default so that it is easy and clear to be consumed by all usrers of the adapter. As a result, necessity of concurrent queue is not obvious.